src/text-input/README.md

```diff
@@ -20,8 +20,10 @@ If the `label` property is not used, we recommend creating a separate `label` an
 ```typescript
 // Basic usage
 w(TextInput, {
```

Comment:
can you search for other mentions of `invalid` in this file and correct them please?

Reply:
I also added a small section on validation to try and make things a bit more clear.